### PR TITLE
Changes PT-BR

### PR DESCRIPTION
**Pull request by bitcoinuser**
## Types of changes

- [ ] fix: bug fixes, e.g. fix Button color on DarkTheme.
- [ ] feat: new features, e.g. add Snackbar component.
- [ ] refactor: code refactor, e.g. new folder structure for components.
- [ ] docs: changes into documentation, e.g. add usage example for Button.
- [ ] test: adding or updating tests, eg unit, snapshot testing.
- [ ] chore: tooling changes, e.g. change circleci config.
- [ ] BREAKING CHANGE: for changes that break existing usage, e.g. change API of a component.

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] Tests for the changes have been added (for bug fixes / features)

* **Other information**:

* **SCREENSHOTS**:

### CODE REVIEW

**Review by patogallaiovlabs — CHANGES_REQUESTED** [review]
I would not change "ativo" to "criptoativo", its pretty obvious that we are talking about crypto since its a wallet, and we would also need to change in the rest of the languages.
Apart from that, I think its a nice PR :)

File: src/common/translations/pt-BR.json [hunk 1]
```diff
@@ -23,14 +23,14 @@
     "wallet": {
       "list": {
         "title": "Sua carteira",
-        "myAssets": "Meus ativos",
-        "allAssets": "Todos os ativos",
-        "addAsset": "Adicionar ativo",
+        "myAssets": "Saldo total",
+        "allAssets": "Todos os criptoativos",
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:27]:
```suggestion
        "allAssets": "Todos os ativos",
```

File: src/common/translations/pt-BR.json [hunk 2]
```diff
@@ -23,14 +23,14 @@
...
+        "addAsset": "Adicionar criptoativo",
         "addWallet": "Adicionar carteira",
         "readOnly": "Somente leitura"
       },
       "history": {
-        "noTransNote": "Não foram encontradas transações realizadas com este ativo.",
+        "noTransNote": "Não foram encontradas transações realizadas com este criptoativo.",
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:33]:
```suggestion
        "noTransNote": "Não foram encontradas transações realizadas com este ativo.",
```

File: src/common/translations/pt-BR.json [hunk 3]
```diff
@@ -78,10 +78,10 @@
         "addReadOnlyWalletNote": "Monitore endereços sem precisar usar as chaves de gasto"
       },
       "selectCurrency": {
-        "title": "Selecionar ativos da carteira"
+        "title": "Selecionar criptoativos da carteira"
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:81]:
```suggestion
        "title": "Selecionar ativos da carteira"
```

File: src/common/translations/pt-BR.json [hunk 4]
```diff
@@ -78,10 +78,10 @@
...
       },
       "selectWallet": {
-        "title": "Selecionar ativo"
+        "title": "Selecionar criptoativo"
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:84]:
```suggestion
        "title": "Selecionar ativo"
```

File: src/common/translations/pt-BR.json [hunk 5]
```diff
@@ -121,11 +121,11 @@
       "walletconnect": {
         "title": "WalletConnect",
         "timeoutTitle": "Tempo expirado",
-        "timeoutBody": "A RWallet não conseguiu estabelecer uma conexão WalletConnect com o dapp dentro do prazo estabelecido. Por favor, tente novamente.",
+        "timeoutBody": "A RWallet não conseguiu estabelecer uma conexão WalletConnect com o dapp dentro do prazo estabelecido. Por favor, tente novamente. Se o problema persistir, exclua todos os dados de navegação (cookies, cache, etc) no navegador que você está usando para acessar o Dapp",
         "timeoutButton": "Ok",
         "emptyWalletError": "Carteira vazia",
-        "selectMainnetAsset": "Selecione uma carteira que tenha pelo menos um ativo na rede principal da RSK",
-        "selectTestnetAsset": "Selecione uma carteira que tenha pelo menos um ativo na rede de testes da RSK",
+        "selectMainnetAsset": "Selecione uma carteira que tenha pelo menos um criptoativo na rede principal da RSK",
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:127]:
```suggestion
        "selectMainnetAsset": "Selecione uma carteira que tenha pelo menos um ativo na rede principal da RSK",
```

File: src/common/translations/pt-BR.json [hunk 6]
```diff
@@ -121,11 +121,11 @@
...
+        "selectTestnetAsset": "Selecione uma carteira que tenha pelo menos um criptoativo na rede de testes da RSK",
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:128]:
```suggestion
        "selectTestnetAsset": "Selecione uma carteira que tenha pelo menos um ativo na rede de testes da RSK",
```

File: src/common/translations/pt-BR.json [hunk 7]
```diff
@@ -138,7 +138,7 @@
         "agreeTo": "Ao clicar em permitir, você concorda em",
         "agreeOne": "- Permitir fornecer o endereço desta carteira",
         "agreeTwo": "- Permitir que seja solicitada a assinatura desta carteira",
-        "assets": "Ativos",
+        "assets": "Criptoativos",
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:141]:
```suggestion
        "assets": "Ativos",
```

File: src/common/translations/pt-BR.json [hunk 8]
```diff
@@ -149,7 +149,7 @@
         "disconnect": "Desconectar",
         "approveAllowance": "Autorizar",
         "approveAllowanceDesc": "Ao clicar em confirmar, você concorda em permitir %{dappUrl} executar contratos inteligentes em seu nome.",
-        "asset": "Ativo",
+        "asset": "Criptoativo",
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:152]:
```suggestion
        "asset": "Ativo",
```

File: src/common/translations/pt-BR.json [hunk 9]
```diff
@@ -213,8 +213,8 @@
         "receiving": "Recebendo"
       },
       "swapSelection": {
-        "title": "Selecionar ativo",
-        "noSwappableCoin": "Desculpe, você não tem nenhum ativo para trocar",
+        "title": "Selecionar criptoativo",
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:216]:
```suggestion
        "title": "Selecionar ativo",
```

File: src/common/translations/pt-BR.json [hunk 10]
```diff
@@ -213,8 +213,8 @@
...
+        "noSwappableCoin": "Desculpe, você não tem nenhum criptoativo para trocar",
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:217]:
```suggestion
        "noSwappableCoin": "Desculpe, você não tem nenhum ativo para trocar",
```

File: src/common/translations/pt-BR.json [hunk 11]
```diff
@@ -229,22 +229,22 @@
         "viewExplorer": "Clique para ver no explorer"
       },
       "addCustomToken": {
-        "title": "Adicionar ativos personalizados",
-        "address": "Endereço do contrato do ativo"
+        "title": "Adicionar criptoativos personalizados",
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:232]:
```suggestion
        "title": "Adicionar ativos personalizados",
```

File: src/common/translations/pt-BR.json [hunk 12]
```diff
@@ -229,22 +229,22 @@
...
+        "address": "Endereço do contrato do criptoativo"
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:233]:
```suggestion
        "address": "Endereço do contrato do ativo"
```

File: src/common/translations/pt-BR.json [hunk 13]
```diff
@@ -229,22 +229,22 @@
...
       },
       "addCustomTokenConfirm": {
-        "title": "Adicionar ativos personalizados",
-        "question": "Você deseja adicionar este ativo?",
-        "token": "Ativo",
-        "symbol": "Símbolo do ativo",
+        "title": "Adicionar criptoativos personalizados",
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:236]:
```suggestion
        "title": "Adicionar ativos personalizados",
```

File: src/common/translations/pt-BR.json [hunk 14]
```diff
@@ -229,22 +229,22 @@
...
+        "question": "Você deseja adicionar este criptoativo?",
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:237]:
```suggestion
        "question": "Você deseja adicionar este ativo?",
```

File: src/common/translations/pt-BR.json [hunk 15]
```diff
@@ -229,22 +229,22 @@
...
+        "token": "Ciptoativo",
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:238]:
```suggestion
        "token": "Ativo",
```

File: src/common/translations/pt-BR.json [hunk 16]
```diff
@@ -229,22 +229,22 @@
...
+        "symbol": "Símbolo do criptoativo",
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:239]:
```suggestion
        "symbol": "Símbolo do ativo",
```

File: src/common/translations/pt-BR.json [hunk 17]
```diff
@@ -229,22 +229,22 @@
...
         "decimals": "Decimais de precisão",
         "balance": "Saldo"
       },
       "addToken": {
-        "title": "Adicionar ativo",
-        "enabledAssets": "Ativos habilitados",
+        "title": "Adicionar criptoativo",
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:244]:
```suggestion
        "title": "Adicionar ativo",
```

File: src/common/translations/pt-BR.json [hunk 18]
```diff
@@ -229,22 +229,22 @@
...
+        "enabledAssets": "Criptoativos habilitados",
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:245]:
```suggestion
        "enabledAssets": "Ativos habilitados",
```

File: src/common/translations/pt-BR.json [hunk 19]
```diff
@@ -229,22 +229,22 @@
...
         "note": "Desative para ocultá-los do seu portfólio.",
-        "addCustomToken": "Adicionar ativo personalizado",
+        "addCustomToken": "Adicionar criptoativo personalizado",
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:247]:
```suggestion
        "addCustomToken": "Adicionar ativo personalizado",
```

File: src/common/translations/pt-BR.json [hunk 20]
```diff
@@ -270,8 +270,8 @@
         "title": "Adicionar carteira somente leitura",
         "address": "Endereço da carteira",
         "note": "Insira um endereço de carteira BTC, RBTC ou nome RNS a ser monitorado.",
-        "question": "Encontramos abaixo os seguintes ativos nesta carteira. Você deseja monitorar este endereço?",
-        "rnsQuestion": "Encontramos abaixo o endereço da carteira e os seguintes ativos. Você deseja monitorar este endereço?",
+        "question": "Encontramos abaixo os seguintes criptoativos nesta carteira. Você deseja monitorar este endereço?",
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:273]:
```suggestion
        "question": "Encontramos abaixo os seguintes ativos nesta carteira. Você deseja monitorar este endereço?",
```

File: src/common/translations/pt-BR.json [hunk 21]
```diff
@@ -270,8 +270,8 @@
...
+        "rnsQuestion": "Encontramos abaixo o endereço da carteira e os seguintes criptoativos. Você deseja monitorar este endereço?",
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:274]:
```suggestion
        "rnsQuestion": "Encontramos abaixo o endereço da carteira e os seguintes ativos. Você deseja monitorar este endereço?",
```

File: src/common/translations/pt-BR.json [hunk 22]
```diff
@@ -306,8 +306,8 @@
         "keys": "Carteiras",
         "anonymousUser": "Usuário anônimo",
         "assets": {
-          "one": "1 Ativo",
-          "other": "{{count}} Ativos"
+          "one": "1 Criptoativo",
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:309]:
```suggestion
          "one": "1 Ativo",
```

File: src/common/translations/pt-BR.json [hunk 23]
```diff
@@ -306,8 +306,8 @@
...
+          "other": "{{count}} Criptoativos"
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:310]:
```suggestion
          "other": "{{count}} Ativos"
```

File: src/common/translations/pt-BR.json [hunk 24]
```diff
@@ -340,12 +340,12 @@
       "keySettings": {
         "title": "Configurações da carteira",
         "walletContains": {
-          "one": "Esta carteira contém 1 ativo",
-          "other": "Esta carteira contém {{count}} ativos"
+          "one": "Esta carteira contém 1 criptoativo",
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:343]:
```suggestion
          "one": "Esta carteira contém 1 ativo",
```

File: src/common/translations/pt-BR.json [hunk 25]
```diff
@@ -340,12 +340,12 @@
...
+          "other": "Esta carteira contém {{count}} criptoativos"
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:344]:
```suggestion
          "other": "Esta carteira contém {{count}} ativos"
```

File: src/common/translations/pt-BR.json [hunk 26]
```diff
@@ -340,12 +340,12 @@
...
         },
         "security": "Segurança",
         "advanced": "Avançado",
-        "Assets": "Ativos",
+        "Assets": "Criptoativos",
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:348]:
```suggestion
        "Assets": "Ativos",
```

File: src/common/translations/pt-BR.json [hunk 27]
```diff
@@ -378,7 +378,7 @@
     },
     "spend": {
       "index": {
-        "title": "Gerenciamento de ativos ao seu alcance",
+        "title": "Gerenciamento de criptoativos ao seu alcance",
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:381]:
```suggestion
        "title": "Gerenciamento de ativos ao seu alcance",
```

File: src/common/translations/pt-BR.json [hunk 28]
```diff
@@ -421,7 +421,7 @@
     "Reset": "Alterar",
     "Finish": "Concluir",
     "Exchange": "Realizar troca",
-    "addAsset": "Adicionar ativo",
+    "addAsset": "Adicionar criptoativo",
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:424]:
```suggestion
    "addAsset": "Adicionar ativo",
```

File: src/common/translations/pt-BR.json [hunk 29]
```diff
@@ -621,7 +621,7 @@
     },
     "readOnlyWallet": {
       "title": "Carteira somente leitura",
-      "body": "Carteiras somente leitura permitem que você monitore ativos na blockchain sem a necessidade de importar as chaves de gasto.\n\nAtualmente são permitidos endereços Bitcoin (SegWit e Legacy), endereços Rootstock ou apelidos RNS."
+      "body": "Carteiras somente leitura permitem que você monitore criptoativos na blockchain sem a necessidade de importar as chaves de gasto.\n\nAtualmente são permitidos endereços Bitcoin (SegWit e Legacy), endereços Rootstock ou apelidos RNS."
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:624]:
```suggestion
      "body": "Carteiras somente leitura permitem que você monitore ativos na blockchain sem a necessidade de importar as chaves de gasto.\n\nAtualmente são permitidos endereços Bitcoin (SegWit e Legacy), endereços Rootstock ou apelidos RNS."
```

File: src/common/translations/pt-BR.json [hunk 30]
```diff
@@ -644,7 +644,7 @@
       "question": "Por que estou vendo esta mensagem?",
       "confirm": "Enviar assim mesmo",
       "cancel": "Voltar e reinserir",
-      "asset": "Ativo",
+      "asset": "Criptoativo",
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:647]:
```suggestion
      "asset": "Ativo",
```

File: src/common/translations/pt-BR.json [hunk 31]
```diff
@@ -23,14 +23,14 @@
...
+        "myAssets": "Saldo total",
         "allAssets": "Todos os ativos",
-        "addAsset": "Adicionar ativo",
+        "addAsset": "Adicionar criptoativo",
         "addWallet": "Adicionar carteira",
         "readOnly": "Somente leitura"
       },
       "history": {
-        "noTransNote": "Não foram encontradas transações realizadas com este ativo.",
+        "noTransNote": "Não foram encontradas transações realizadas com este criptoativo.",
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json]:
```suggestion
        "noTransNote": "Não foram encontradas transações realizadas com este ativo.",
```

File: src/common/translations/pt-BR.json [hunk 32]
```diff
@@ -23,9 +23,9 @@
     "wallet": {
       "list": {
         "title": "Sua carteira",
-        "myAssets": "Meus ativos",
+        "myAssets": "Saldo total",
         "allAssets": "Todos os ativos",
-        "addAsset": "Adicionar ativo",
+        "addAsset": "Adicionar criptoativo",
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:28]:
```suggestion
        "addAsset": "Adicionar ativo",
```

File: src/common/translations/pt-BR.json [hunk 33]
```diff
@@ -121,7 +121,7 @@
       "walletconnect": {
         "title": "WalletConnect",
         "timeoutTitle": "Tempo expirado",
-        "timeoutBody": "A RWallet não conseguiu estabelecer uma conexão WalletConnect com o dapp dentro do prazo estabelecido. Por favor, tente novamente.",
+        "timeoutBody": "A RWallet não conseguiu estabelecer uma conexão WalletConnect com o dapp dentro do prazo estabelecido. Por favor, tente novamente. Se o problema persistir, exclua todos os dados de navegação (cookies, cache, etc) no navegador que você está usando para acessar o Dapp",
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:124]:
```suggestion
        "timeoutBody": "A RWallet não conseguiu estabelecer uma conexão WalletConnect com o dapp dentro do prazo estabelecido. Por favor, tente novamente. Se o problema persistir, exclua todos os dados de navegação (cookies, cache, etc) no navegador que você está usando para acessar o dapp",
```

**Comment by bitcoinuser** [comment]
> I would not change "ativo" to "criptoativo", its pretty obvious that we are talking about crypto since its a wallet, and we would also need to change in the rest of the languages.
> Apart from that, I think its a nice PR :)

Done.

**Review by patogallaiovlabs — APPROVED** [review]
LGTM

**Pull request merged**